languages/ruby/exercises/concept/instance-variables/.docs/hints.md

```diff
@@ -0,0 +1,39 @@
+## General
+
+About initializing object instances:
+
+- [Ruby Guides: Initialize Method][rg-initialize-method]
+
+About instance variables:
+
+- [Ruby For Beginners: Instance variables][rfb-instance-variables]
+- [Ruby Guides: Instance variables][rg-instance-variables]
+- [Ruby User's Guide: Instance variables][rug-instance-variables]
+- [Geeks for Geeks: Ruby Getters and Setters Methods][gfg-getter-setters]
+
+## 1. Make new attendees
+
+- Using the references, complete the initialize method to save the height to the object's state.
+
+## 2. How tall is the attendee
+
+- Create a getter for the saved height state.
+
+## 3. What is the ride pass's id
+
+- Create a getter for the pass id.
+- It is okay and expected to return `nil` before a pass is issued to the `Attendee`.
+
+## 4. Allow people to buy a pass
+
+- Using the `issue_pass!` setter method set the instance's state to the argument.
```

Comment:
```suggestion
- Using the `issue_pass!` method, set the instance's state to the argument.
```

Reasoning:

A writer method has the pattern of `name=(argument)` with the equal sign as the last character.  The macros are 3:

`_reader`, `_writer` and `_accessor` which includes the first two.  So the term we would use in Ruby would be a "writer" rather than the "comp-sci" term of "setter".